.github/workflows/staging-lint-checks.yaml

```
@@ -0,0 +1,71 @@
+name: Staging Lint Checks
+
+on:
+  pull_request:
+    types:
+      # default
+      - opened
+      - synchronize
+      - reopened
+      # body can be used to select rule(s) for testing
+      - edited
+      # labels can be used to select rule(s) for testing
+      - labeled
+      - unlabeled
+
+env:
+  SPEC_REPO: Azure/azure-rest-api-specs
+  SPEC_CHECKOUT_PATH: specs
+  FAIL_ON_ERRORS: "false" # Set to "true" to fail the job when error findings appear
+  MAX_FILES: "100"
+  ALLOWED_RPS: "compute,monitor,sql,hdinsight,network,resource,storage"
+
+jobs:
+  run-selected-rules:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-24.04
```

More reliable to pin to a version, and upgrade once every 2-4 years when we want.

---

makes sense.